<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>631</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="ptLog"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="leChat">
      <property name="placeholderText">
       <string>chat</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pbSend">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>631</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="leIp">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
      <property name="placeholderText">
       <string>ip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lePort">
      <property name="text">
       <string>9999</string>
      </property>
      <property name="placeholderText">
       <string>port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leName">
      <property name="text">
       <string>test</string>
      </property>
      <property name="placeholderText">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbConnect">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
